rust_icu_ucsdet/src/lib.rs

```diff
@@ -0,0 +1,274 @@
+// Copyright 2023 Colerar
```

Comment:
Hi. Sorry - but it seems that we can't accept this copyright notice as written.

Note, you *do* retain the copyright for the things you authored, and the source of truth is the version control history.

Just that our in-house counsel requires the notice to be expressed in a different way. *And* a project needs an approval to change the way the notice is expressed.

So I'm letting you know early in case you want to either (1) make the change as above; or (2) wait until we get an approval to change the copyright claim method.

An easy way around is to add the regular Google copyright notice, and also add a file AUTHORS listing Google LLC and yourself. You can then file a bug for me to get an approval to modify the copyright to refer to that file.